pcdsdevices/smarpod.py

```diff
@@ -0,0 +1,133 @@
+"""
+Module for the SmarPod and related devices.
+"""
+from ophyd import PVPositioner
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+
+class SmarPod(PVPositioner):
```

Comment:
Have you given this a try, @patoppermann? Like running it in Python and instantiating a `SmarPod` instance?

I ask because `PVPositioner` requires some components to be defined, which I don't see here: https://confluence.slac.stanford.edu/display/PCDS/How+to+Create+an+Ophyd+Device#HowtoCreateanOphydDevice-PVPositioner

`PVPositioner` should raise an exception if those are not set: see [here](https://github.com/bluesky/ophyd/blob/master/ophyd/pv_positioner.py#L118)

I'm not sure if a single `PVPositioner` will work for the SmarPod, since it has 6 DOF (x/y/z/rx/ry/rz) to check. Normally `PVPositioner` works with scalars, but _maybe_...